.github/workflows/regression_tests.yml

```diff
@@ -33,6 +33,17 @@ jobs:
           path: cloudsc
           ref: develop
 
+      - name: Clone ECWAM
+        run: |
+          mkdir ${{ github.workspace }}/ecwam
+          mkdir -p ~/.ssh/ && touch ~/.ssh/known_hosts
+          cat << 'EOF' > ~/.ssh/known_hosts
+          ${{ secrets.BITBUCKET_HOSTKEY }}
+          EOF
+          eval `ssh-agent -s`
+          ssh-add - <<< '${{ secrets.ECWAM_REPOSITORY_SSH_KEY }}'
+          git clone --branch naan-phys-gpu ssh://git@git.ecmwf.int/~nawd/ecwam-bundle.git ${{ github.workspace }}/ecwam
```

Comment:
Uhhhhh, does ecWAM not self-bundle? Seems pretty back-bending to go back to BITBUCKET just for the bundle. Do we really need this, or could this be fixed upstream in github to avoid this?

Reply:
No ecWam does not self-bundle. I don't think Willem would be opposed to that, I can speak to him about including the bundle in the ecwam repo, or as a middle-ground to have the bundle on github so we don't need to go to bitbucket.

Reply:
I believe this has now been rectified and updated on ecWAM side. So all good on this one!